dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -126,50 +126,22 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
                 m_registering = false;
             }) };
 
-            winrt::guid storedComActivatorGuid{ GUID_NULL };
-            if (!PushNotificationHelpers::IsPackagedAppScenario())
+            winrt::guid registeredClsid{};
+            if (AppModel::Identity::IsPackagedProcess())
```

Comment:
There is a missing scenario for Packaged Apps here. If it is a Packaged App running as UnpackagedScenario(due to Push), we need to create the correct mapping with LRP. Please sync with @pmpurifoy on details here